README.md

```diff
@@ -1,2 +1,91 @@
+
+## Prerequisites
+
+Before you begin, ensure you have the following installed:
+
+- A working Open edX instance using the Tutor platform.
+
+## Installation Steps
+
+Follow the steps below to install the Forum app in your Tutor environment:
+
+### 1. Add a Forum App to the openedX dependencies
+
+To add this package to the Open edX dependencies, include it in the `OPENEDX_EXTRA_PIP_REQUIREMENTS`. Since the package is not yet available on PyPI, we will install it directly from the Git repository.
+
+Run the following command to append the package to your Tutor configuration:
+
+```bash
+tutor config save --append 'OPENEDX_EXTRA_PIP_REQUIREMENTS=git+https://github.com/edly-io/forum.git@master'
+```
+
+### 2. Save config and rebuild open-edX Image
+
+```yaml
+tutor config save
+tutor images build openedx
+```
+
+### 3. Run Launch
+
+Run the launch commad. It is necessary for building the migrations present in the Forum app. 
+
+```bash
+# Dev
+tutor dev launch
+```
+
+### 4. Accessing the Forum
+
+There are no changes needed in the [Discussion MFE](https://github.com/openedx/frontend-app-discussions). You should see the forum interface where users can start discussions, post comments, and interact with each other. In the backend, the API calls are made via the Forum app instead of the CS Comments Service.
+
+### 5. Development (For Developers)
+
+In certain scenarios, you may need to mount the Forum package for extending or debugging features. For this, you need to install the [forumv2](https://gist.github.com/taimoor-ahmed-1/9e947a06d127498a328475877e41d7c0) plugin. Follow these steps to accomplish this:
+
+1. **Clone the forum repo:**
+    ```bash
+    git clone git@github.com:edly-io/forum.git
+    ```
+2. **Mount the repo:**    
+    ```bash
+    tutor mounts add path/to/forum/repo
+    ```
+    
+3. **Install this plugin:**    
+    ```bash
+    tutor plugins install https://gist.githubusercontent.com/taimoor-ahmed-1/9e947a06d127498a328475877e41d7c0/raw/6152bdc312f941e79d50e2043f00d3d059de70a7/forum-v2.py
+
+    ```
+
+4. **Enable the plugin:**
+    
+    ```bash
+    tutor plugins enable forumv2
+    ```
+
+5. **Save Changes:**
+    
+    After the plugin is enabled, further changes to the plugin code are applied with:
+    ```bash
+    tutor config save
+    ```
+
+6. **Build the openedx-dev Docker image:**
+    ```bash
+    tutor images build openedx-dev
+    ```
+
+7. **Launch the platorm:**
+    ```bash
+    tutor dev launch
+    ```
+
+## License
+
+This project is licensed under the MIT License.
```

Comment:
You can also copy these datails from [README.rst](https://github.com/edly-io/forum/blob/master/README.rst#:~:text=Getting%20Help%20page.-,License,are%20expected%20to%20follow%20the%20Open%20edX%20Code%20of%20Conduct.,-People).

Reply:
I think there should be only one README file. It's better to delete any one of it.

Reply:
Yes, we can delete one.

And, we already have a [LICENSE.txt](https://github.com/edly-io/forum/blob/master/LICENSE.txt). You can also link that in the readme. Contributing and The Open edX Code of Conduct can also be added.